app/build.gradle

```diff
@@ -115,6 +115,9 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-common-java8:2.1.0-alpha02'
     implementation "android.arch.navigation:navigation-fragment:$navigation_version"
     implementation "android.arch.navigation:navigation-ui-ktx:$navigation_version"
+    implementation 'com.adjust.sdk:adjust-android:4.17.0'
+    implementation 'com.android.installreferrer:installreferrer:1.0'
+    implementation 'com.google.android.gms:play-services-analytics:16.0.7'
```

Comment:
😞

Reply:
why the sad face?? 🤔

Reply:
google play services! :p

Reply:
Should I take this out?

Reply:
no, it's required for adjust. with it, however, we can't release into app stores other than the google play store (e.g. F-droid).